Add installation instruction (#9)
swoolcock/rush: README.md

## Status
WIP but playable.

## Installation

> Prebuilt binary releases are available if you're looking for testing the ruleset or don't possess a developpment environment:

Reviewer: ```suggestion
Prebuilt binary releases are available if you're looking for testing the ruleset without needing a development environment set up in place:
```
How does this one sound?

Reviewer 2: ```
Prebuilt binary releases are available if you are looking to test or play and do not wish to compile yourself.
```

| [Releases Page](https://github.com/swoolcock/rush/releases/) |
| -------------|

> **NOTE**: For the moment, only desktop platforms (Windows, mac OSX, Linux) are supported.

Reviewer: ```suggestion
**NOTE**: osu!lazer runs custom rulesets only on desktop platforms (Windows, macOS, Linux) for the time being.
```
Better provide **what** is not supporting to avoid confusion.

> The ruleset consists of a single DLL file that you'll need to drop in the `rulesets` directory of osu!lazer data directory.

Reviewer: ```suggestion
The ruleset consists of a single DLL file that you'll need to drop it in the `rulesets` directory inside the osu!lazer data directory.
```

Reviewer 2: @Game4all 's is more grammatically correct.

### Installation instructions

> 1. Navigate to the osu!lazer data directory. You can navigate to it by opening the osu! settings panel and using "open osu! folder" button. Alternatively you can directly navigate to the `rulesets` folder via your OS directory explorer at the following locations:

Reviewer: ```suggestion
1. Navigate to the osu!lazer data directory. You can navigate to it by opening the settings panel on osu!lazer and clicking on the "open osu! folder" button. Alternatively you can directly navigate to the `rulesets` directory via your OS directory explorer at the following locations:
```

> 1. Navigate to the osu!lazer data directory. You can navigate to it by opening the settings panel on osu!lazer and clicking on the "open osu! folder" button. Alternatively you can directly navigate to the `rulesets` directory via your OS directory explorer at the following locations:

Reviewer 2: Replace `You can navigate to it by` with `You can do so by`, and replace `on osu!lazer` with `in osu!lazer`.

    - `%AppData%/osu/rulesets` on Windows
    - `~/.local/share/osu/rulesets` on Linux / mac OSX 

>     **NOTE**: These paths may be incorrect if you have relocated your osu! data directory to another directory.

Reviewer: ```suggestion
    **NOTE**: If you have relocated your osu! data directory to another directory, the `rulesets` directory will be there instead.
```

> 2. Drag and drop into the `rulesets` directory the ruleset DLL file.

Reviewer: ```suggestion
2. Drag and drop the ruleset's DLL file into the `rulesets` directory.
```

3. Have fun!

>   If lazer was running while installing the ruleset, you may need to restart the game in order for the ruleset to appear in-game.

Reviewer: ```suggestion
  If osu!lazer was running while installing the ruleset, you may need to restart it in order for the ruleset to be acknowledged and loaded for playing.
```

Reviewer 2: In @Game4all 's original sentence, just change `lazer` to `osu!lazer` and remove `in-game`.

> **NOTE**: rulesets don't automatically update alongside lazer but have a compatibility mechanism to continue using them with newer lazer versions. However, some changes made on game-side may break that compatibility and require installing a newer version of the ruleset.

Reviewer: ```suggestion
**NOTE**: Custom Rulesets do not automatically update alongside osu!lazer but have a compatibility mechanism to continue using them on newer game versions. However, some changes made game-side may break that compatibility and require installing a newer version of the ruleset.
```

> Thus it is recommended that you periodically head over the releases page and replace the ruleset DLL in the `rulesets` folder with the latest available version.

Reviewer: ```suggestion
Thus it is recommended that you periodically head over the releases page and replace the ruleset DLL in the `rulesets` directory with the latest available version.
```
